Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Sadly I would argue Apple has abdicated its responsibility here 😭.

Nonetheless, back to trying to figure out how to least break all the things with `node` I guess 😄.